examples/gno.land/p/moul/md/md.gno

```diff
@@ -255,7 +255,16 @@ func EscapeText(text string) string {
 // Columns returns a formatted row of columns using the Gno syntax.
 // If you want a specific number of columns per row (<=4), use ColumnsN.
 // Check /r/docs/markdown#columns for more info.
-func Columns(contentByColumn []string) string {
+func Columns(contentByColumn []string, padded bool) string {
```

Comment:
Maybe specify the difference using `padded` in the godoc?

Reply:
```suggestion
// If padded=true & the final <gno-columns> tag is missing column content, an empty
// column element will be placed to keep the cols per row constant.
// Padding works only with colsPerRow > 0.
func Columns(contentByColumn []string, padded bool) string {
```